### ISSUE TITLE
weird behavior in `tmux` when paste `('` from clipboard

### ISSUE THREAD

**Issue by tonytonyjan** [fish-shell/fish-shell#7831]
## Steps to Reproduce

TLDR: https://i.imgur.com/jGCOtNW.mp4

1. copy `('`
2. open Terminal.app
3. run `tmux`
4. in `tmux` pane, use `cmd` + `v` to paste `('`

## Expected

![](https://i.imgur.com/uuh5I44.png)

## Actual

![](https://i.imgur.com/Wqarh9I.png)

- A wearied `^[[201` shows up.
- The behaviour is not consistent, in most of time it shows `^[[201`, and sometimes not.

## Environment

```
$ fish -v 
fish, version 3.2.0
$ tmux -V
tmux 3.1c 
$ uname -a
Darwin Skoll 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64
$ echo $TERM
xterm-256color
$ cat ~/.tmux.conf 
set -g status-position top
bind % split-window -h -c "#{pane_current_path}"
bind '"' split-window -c "#{pane_current_path}"
bind c new-window -c "#{pane_current_path}"
$ ls ~/.config/fish/
fish_variables
```

![terminal version](https://i.imgur.com/pzxLsyt.png)

**Comment by zanchey** [comment]
Duplicate of #7770

**Comment by zanchey** [comment]
This will be fixed in the upcoming 3.2.1 point release.

**Comment by tonytonyjan** [comment]
@zanchey Thank you!
